php_sql/task1/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title> Форми </title>
+</head>
+
+<body>
+    <form method="post" action="index.php">
+        <fieldset>
+            <legend> Информация за избираемата дисциплина </legend>
+            <label for="courseTitle">Име на курса:</label>
+            <input id="coursetitle" name="title" require> </br>
```

Comment:
The keyword is "required" not "require". Please, fix everywhere. 
https://www.w3schools.com/tags/att_input_required.asp